src/Components/Components/src/Routing/RouteTableFactory.cs

```diff
@@ -34,26 +34,53 @@ public static RouteTable Create(Assembly appAssembly)
 
         internal static RouteTable Create(IEnumerable<Type> componentTypes)
         {
-            var routes = new List<RouteEntry>();
-            foreach (var type in componentTypes)
+            var templatesByHandler = new Dictionary<Type, string[]>();
```

Comment:
`templatesByComponent`?

Reply:
It’s weird, but I’m using the term “handler” for consistency with the rest of the code in this file.